packages/documentation-v7/src/stories/components/card/card.demo.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import type { StoryObj } from '@storybook/web-components';
+import meta, { Default } from './card.stories';
+
+export default {
+  ...meta,
+  title: 'Hidden/demos/components/Card',
+};
+
+type Story = StoryObj;
+
+export const BackgroundImage: Story = {
+  ...Default,
+  parameters: {
+    controls: {
+      include: [],
+    },
+  },
```

Comment:
Is it necessary?

Reply:
This config removes all controls, it is easier then using `exclude` and listing everything.
We can remove it when going for "docs only".